source/release-notes/backend.txt

```diff
@@ -13,6 +13,21 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240124:
+
+24 January 2024 Release
+~~~~~~~~~~~~~~~~~~~~~~~
+
+- Included timezones in the timestamps for log messages, deployment histories, and more in the App Services UI.
+  
+- GET methods for the App Services Admin API now return full endpoints instead of partial endpoints.
+
+- Raised the default websocket read limit from 16 MB to 20 MB. Added an app-level config setting to raise the read limit if needed.
+
+- Added an App Services Admin API endpoint to trigger a client reset.
```

Comment:
I'm still not super familiar with BAAS RNs - should there be more info here?

Reply:
Would like to add link to Admin API endpoint reference when the corresponding entry is created.